mobile/src/app/components/resources/resources.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-resources',
+  templateUrl: './resources.component.html',
+  styleUrls: ['./resources.component.scss'],
+})
+export class ResourcesComponent implements OnInit {
+  data: ArrayResource = [
+    {'General Information': [{ infoName: 'What to expect at a protest', url: 'https://www.law.nyu.edu/centers/race-inequality-law/protest-tips' }]},
+    {'Find Legal Resources': [{ infoName: 'NGL Legal Support Hotline', url: 'https://www.communityjusticeexchange.org/nbfn-directory' }, { infoName: 'National Bail Fund Network', url: 'https://www.nlg.org/massdefenseprogram/' }]},
+    {'Support us': [{ infoName: 'GoFundMe', url: 'https://www.gofundme.com/f/help-build-a-protest-app' }]}
+  ]
+
+  constructor() { }
+  ngOnInit() {}
+
+  //delete later if unused and utilising ionic button href
+  redirectTo(url){
+    console.log(url)
+    window.location.href=url
+  }
+}
+
+//types and interfaces
```

Comment:
Not really a necessary comment.

It's easy to see that these are type and interfaces, but it's more useful to explain what they're doing or what the values are for. I'll follow up on this comment with an example soon.

Reply:
So using JSDoc, you could do something like

```
/** Associates a list of resources to their related resource section */
type Resource = Record<string, Array<ResourceDesc>>
/** A list of resources */
type ArrayResource = Array<Resource>


/** Defines an individual resource item on the page */
interface ResourceDesc {

  /** The name that will be displayed for the item */
  infoName: string

  /** The url redirect for the item */
  url: string
}

```